schemas/test/en/test_question_definition.json

```diff
@@ -43,31 +43,10 @@
                                 "id": "question",
                                 "title": "Do you connect a LiFePO4 battery to your <em>photovoltaic system</em> to store surplus energy?",
                                 "type": "General",
-                                "definitions": [
-                                    {
-                                        "title": "What is a photovoltaic system?",
-                                        "contents": [
-                                            {
-                                                "description": "A typical photovoltaic system employs solar panels, each comprising a number of solar cells, which generate electrical power. PV installations may be ground-mounted, rooftop mounted or wall mounted. The mount may be fixed, or use a solar tracker to follow the sun across the sky."
-                                            }
-                                        ]
-                                    },
-                                    {
-                                        "title": "Why use LiFePO4 batteries?",
-                                        "contents": [
-                                            {
-                                                "title": "3 Benefits of LifePO4 batteries."
-                                            },
-                                            {
-                                                "list": [
-                                                    "LifePO4 batteries have a life span 10 times longer than that of traditional lead acid batteries. This dramatically reduces the need for battery changes.",
-                                                    "Lithium iron phosphate batteries operate with much lower resistance and consequently recharge at a faster rate.",
-                                                    "LifeP04 lightweight batteries are lighter than lead acid batteries, usually weighing about 1/4 less."
-                                                ]
-                                            }
-                                        ]
-                                    }
-                                ],
+                                "definition": {
```

Comment:
https://user-images.githubusercontent.com/108286538/233354873-b19ddc94-d923-4f4d-8a65-36bf4b7c1158.mov

I can't seem to get this definition to show, not sure if I'm misunderstanding how it works or what exactly the behaviour should be

Reply:
Thats strange it shows up for me.
<img width="771" alt="image" src="https://user-images.githubusercontent.com/113098086/233380533-cfd264d7-354a-445c-936b-0c3de546162e.png">

Reply:
Yeah works for me too. I think there is something in Kyles local environment that needs updating

Reply:
Yep my bad, didn't update DS templates 🙃